api/src/main/java/org/apache/xtable/model/storage/OneDataFilesDiff.java

```diff
@@ -31,7 +31,7 @@
 @Value
 @EqualsAndHashCode(callSuper = true)
 @SuperBuilder
-public class OneDataFilesDiff extends DataFilesDiff<OneDataFile, OneDataFile> {
+public class OneDataFilesDiff extends DataFilesDiff<InternalDataFile, InternalDataFile> {
```

Comment:
Update the name of this to `InternalDataFilesDiff`?

Reply:
My understanding is that the "internal" prefix will be used selectively, in cases where the class name collision across packages is high. DataFileDiff is not a common class name. Accordingly, I have renamed the base file, old DataFilesDiff -> new FilesDiff, and renamed OneDataFilesDiff to DataFilesDiff. 
WDYT?

Reply:
Sounds good to me

Reply:
I have pushed DataFilesDiff rename chanegs in this PR.